TLM/TLM/Traffic/Impl/SegmentEnd.cs

```diff
@@ -1,287 +1,357 @@
-#define DEBUGFRONTVEHx
-#define DEBUGREGx
+#define DEBUGFRONTVEH_off
+#define DEBUGREG_off
 #define DEBUGMETRIC
-#define DEBUGMETRIC2x
-
-using System;
-using System.Collections.Generic;
-using ColossalFramework;
-using TrafficManager.Geometry;
-using TrafficManager.TrafficLight;
-using TrafficManager.Custom.AI;
-using TrafficManager.Util;
-using System.Threading;
-using TrafficManager.State;
-using TrafficManager.UI;
-using TrafficManager.Manager;
-using System.Linq;
-using CSUtil.Commons;
-using TrafficManager.Geometry.Impl;
-using TrafficManager.Manager.Impl;
-using TrafficManager.Traffic.Data;
-using TrafficManager.Traffic.Enums;
-
-/// <summary>
-/// A segment end describes a directional traffic segment connected to a controlled node
-/// (having custom traffic lights or priority signs).
-/// </summary>
+#define DEBUGMETRIC2_off
```

Comment:
These don't seem to be used in this file?

Reply:
Commented out. I'm renaming `#define SOMETHINGx` to `#define SOMETHING_off`